src/Schema/SQLiteSchemaManager.php

```diff
@@ -69,7 +69,7 @@ public function dropForeignKey(string $name, string $table): void
 
         $foreignKey = $table->getForeignKey($name);
 
-        $this->alterTable(new TableDiff($table, [], [], [], [], [], [], [], [], [], [], [$foreignKey]));
+        $this->alterTable(new TableDiff($table, modifiedForeignKeys: [$foreignKey]));
```

Comment:
Probably meant to be this instead 
```suggestion
        $this->alterTable(new TableDiff($table, droppedForeignKeys: [$foreignKey]));
```